lms/djangoapps/grades/scores.py

```diff
@@ -89,7 +89,12 @@ def get_score(user, block, scores_client, submissions_scores_cache, weight, poss
         if possible is None:
             possible = score.total
         elif possible != score.total:
-            log.error(u"Persisted Grades: possible value {} != score.total value {}".format(possible, score.total))
+            log.error(
+                u"Persistent Grades: scores.get_score, possible value {} != score.total value {}".format(
```

Comment:
Note that these both now also specify `<thing>.<otherthing>`, this is what makes them fit nicely into the report linked above.